src/functionBased/components/TodosList.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable */
```

Comment:
I think disabling eslint in the entire file will make it impossible for you to follow the guidelines required for the file. I understand some rules are hard to implement but please try to make sure you are not disabling eslint in the entire file.